### ISSUE TITLE
Have difficulty in reproducing the sota result

### ISSUE THREAD

**Issue by Pexure** [KentoNishi/Augmentation-for-LNL#5]
Hi,
Thank you for your great work. I tried to reproduce c100/sym90/AugDesc-WS, but failed twice with two differently generated noise labels. The sign is the test acc gets stuck at ~40 and wouldn't go up. Do you have any ideas about that?

**Comment by KentoNishi** [comment]
Hello, thanks for reaching out!

You mentioned that you ran `c100.90sym.AugDesc-WS`. The expected accuracy for this benchmark is roughly 40%. Did you perhaps mean to run the c10 experiment instead?

**Comment by Pexure** [comment]
Sry, you are right. I mean `c10.90sym.AugDesc-WS`. The test acc hang around ~40.

**Comment by KentoNishi** [comment]
As mentioned in the README, results may be unstable at particularly high noise ratios (ex. 90% on CIFAR-10). This is an issue that we have not been able to resolve as of yet.

There are some things which may be worth trying, including:
* Using the `RandAugment` strategy instead of `AutoAugment`
* [Fixing a small bug as noted in our code](https://github.com/KentoNishi/Augmentation-for-LNL/blob/908a22b23ca4cd5c70c52bdaa6a4304a5cbb1fda/dataloader_cifar.py#L475-L480)
* Re-running the experiment after adjusting some hyperparameters

Please let me know if/when you are able to successfully reproduce our results!

**Comment by cp3cp** [comment]
Hello, thank you for your good work and I have the same problem about this issue. I used RandAugment strategy but the accuracy is still far away from 90%. It gets stuck at around 50%. So I wanna ask if you can public the pre-train model? Or do you have any suggestion about this now? Thanks in advance~

**Comment by KentoNishi** [comment]
I've attached my pre-trained models from 2 years ago below. One is 90% CIFAR-10 with AutoAugment, the other with RandAugment. Please let me know if you would like to have access to any other pretrained models.

https://www.file.io/7eSA/download/3rUEGYKjs39J

My current research direction is different from this line of work, so I don't have any new suggestions other than to try again. The higher noise ratios exhibit a lot of volatility due to the randomness of augmentation, and my biggest regret with this paper is that I did not investigate this instability further. If anyone has any findings, please let me know!

**Comment by cp3cp** [comment]
wow that's great! Thanks a lot! I am new about LNL and still trying to reproduce the work so that will be helpful to me.
